polling_stations/apps/data_collection/management/commands/import_south_gloucestershire.py

```diff
@@ -1,68 +1,53 @@
-from django.contrib.gis.geos import Point
 from data_collection.management.commands import BaseXpressDemocracyClubCsvImporter
 
 
 class Command(BaseXpressDemocracyClubCsvImporter):
     council_id = "E06000025"
-    addresses_name = "europarl.2019-05-23/Version 1/Democracy_Club__23May2019.tsv"
-    stations_name = "europarl.2019-05-23/Version 1/Democracy_Club__23May2019.tsv"
-    elections = ["europarl.2019-05-23"]
+    addresses_name = "parl.2019-12-12/Version 1/Democracy_Club__12December2019sglo.tsv"
+    stations_name = "parl.2019-12-12/Version 1/Democracy_Club__12December2019sglo.tsv"
+    elections = ["parl.2019-12-12"]
     csv_delimiter = "\t"
+    allow_station_point_from_postcode = False
 
     def station_record_to_dict(self, record):
         rec = super().station_record_to_dict(record)
 
-        # Trust Hall-Stoke Gifford
-        if record.polling_place_id == "10592":
-            rec["location"] = Point(-2.540965, 51.516655, srid=4326)
+        # # Trust Hall-Stoke Gifford
+        # if record.polling_place_id == "10592":
+        #     rec["location"] = Point(-2.540965, 51.516655, srid=4326)
```

Comment:
This should be deleted if it's no longer relevant.

Reply:
I'd left this in in case they ever used that location again, but can certainly remove it.